guides/en/for.md

```diff
@@ -46,8 +46,14 @@ The above example would output `appleorangebanana`
 
 ### For In Loop (over a query) CF10+
 
-    for (row in query) {
-        writeOutput(query.currentRow);
+    fruits = queryNew("name", "varchar", [
+        ["apple"],
+        ["banana"],
+        ["orange"]
+    ]); 
+
+    for (item in fruits) {
```

Comment:
I would recommend keeping the name `row` instead of `item`.

Reply:
Yes, I agree.